server/server.go

```diff
@@ -42,6 +42,10 @@ func getLanguages(w http.ResponseWriter, req *http.Request, config Config) {
 	username := query.Get("user")
 	queryLimit := query.Get("limit")
 
+	provider := "github"
+
+	provider = query.Get("provider")
```

Comment:
```go
provider := "github"
provider = query.Get("provider")
```

Is the same of 
```go
provider :=  query.Get("provider")
```

isn't ?

Reply:
I don't think so. When speaking about the server, I think it doesn't get the default flags from cobra

Reply:
but if it doesn't get the values from cobra the variable would be replace to empty string then. 
I'm going to check this anyway.